docs/cluster_up_down.md

```diff
@@ -315,11 +316,31 @@ To see metrics in the web console, you must first browse to the Hawkular metrics
 | ---- |
 | This feature requires an oc command v1.4 or newer |
 
-You can install logging aggregation components by specifying the --logging argument when invoking `oc cluster up`.
+You can install logging aggregation components by specifying the `--logging` argument when invoking `oc cluster up`.
 
 With logging aggregation installed, a new link will appear in the logs tab of a running pod in the web console.
 
 
+## Installing the Service Catalog
+
+| NOTE |
+| ---- |
+| This feature requires an oc command v3.6 or newer |
+
+You can enable the service catalog component by specifying the `--service-catalog` argument when invoking `oc cluster up`.
+
+Enabling the service catalog has the following effect:
+
+1. The API aggregator is enabled to provide a unified API endpoint for both OpenShift/Kubernetes resources and the new APIs introduced by the service catalog.
+1. A service catalog deployment is created to deploy and run the service catalog.
+1. The template broker is enabled in the OpenShift master server and registered with the service catalog.
+1. The web console is configured to use the new service catalog landing page.
+
+Although the template broker is enabled, by default it will reject all API requests. Because the service catalog is not yet able to properly authenticate to OpenShift api endpoints, it is necessary to disable the authentication requirement for the template broker APIs. Once startup is complete, `oc cluster up` will output instructions for disabling the authentication, however this step should only be undertaken after fully understanding the security ramifications of allowing anyone who see your cluster, to be able to provision templates into any project, as any user.
```

Comment:
How about:

On completion `oc cluster up` will output a command that you may run if you want to use the template broker with the service catalog.  CAUTION: running this command has significant adverse security effects as it disables authenticated access to the template broker.  This allows anyone who can access your master to provision templates into any project as any user.